contrib/yolov5/tests/test_torchhub.py

```diff
@@ -0,0 +1,42 @@
+#  Copyright (c) 2021 Rikai Authors
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+from pathlib import Path
+
+from pyspark.sql import SparkSession
+
+from rikai.contrib.yolov5.transforms import OUTPUT_SCHEMA
+
+
+def test_yolov5(spark: SparkSession):
+    work_dir = Path().absolute().parent.parent
+    image_path = f"{work_dir}/python/tests/assets/test_image.jpg"
+    spark.sql(
+        f"""
+CREATE MODEL yolov5
+FLAVOR yolov5
+PREPROCESSOR 'rikai.contrib.yolov5.transforms.pre_processing'
+POSTPROCESSOR 'rikai.contrib.yolov5.transforms.post_processing'
+OPTIONS (device="cpu", batch_size=32)
+RETURNS {OUTPUT_SCHEMA}
+USING "torchhub:///ultralytics/yolov5:v6.0/yolov5s";
```

Comment:
This why I'm using `yolov5:v6.0`, seems to be a yolov5 bug.
```
(rikai) ➜  python git:(feature/torchhub) python
Python 3.8.10 (default, May 19 2021, 11:01:55)
[Clang 10.0.0 ] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import torch
>>> torch.hub.load("ultralytics/yolov5:v5.0", "yolov5s")
Using cache found in /Users/da/.cache/torch/hub/ultralytics_yolov5_v5.0

                 from  n    params  module                                  arguments
  0                -1  1      3520  models.common.Focus                     [3, 32, 3]
  1                -1  1     18560  models.common.Conv                      [32, 64, 3, 2]
  2                -1  1     18816  models.common.C3                        [64, 64, 1]
  3                -1  1     73984  models.common.Conv                      [64, 128, 3, 2]
  4                -1  1    156928  models.common.C3                        [128, 128, 3]
  5                -1  1    295424  models.common.Conv                      [128, 256, 3, 2]
  6                -1  1    625152  models.common.C3                        [256, 256, 3]
  7                -1  1   1180672  models.common.Conv                      [256, 512, 3, 2]
  8                -1  1    656896  models.common.SPP                       [512, 512, [5, 9, 13]]
  9                -1  1   1182720  models.common.C3                        [512, 512, 1, False]
 10                -1  1    131584  models.common.Conv                      [512, 256, 1, 1]
 11                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']
 12           [-1, 6]  1         0  models.common.Concat                    [1]
 13                -1  1    361984  models.common.C3                        [512, 256, 1, False]
 14                -1  1     33024  models.common.Conv                      [256, 128, 1, 1]
 15                -1  1         0  torch.nn.modules.upsampling.Upsample    [None, 2, 'nearest']
 16           [-1, 4]  1         0  models.common.Concat                    [1]
 17                -1  1     90880  models.common.C3                        [256, 128, 1, False]
 18                -1  1    147712  models.common.Conv                      [128, 128, 3, 2]
 19          [-1, 14]  1         0  models.common.Concat                    [1]
 20                -1  1    296448  models.common.C3                        [256, 256, 1, False]
 21                -1  1    590336  models.common.Conv                      [256, 256, 3, 2]
 22          [-1, 10]  1         0  models.common.Concat                    [1]
 23                -1  1   1182720  models.common.C3                        [512, 512, 1, False]
 24      [17, 20, 23]  1    229245  models.yolo.Detect                      [80, [[10, 13, 16, 30, 33, 23], [30, 61, 62, 45, 59, 119], [116, 90, 156, 198, 373, 326]], [128, 256, 512]]
Model Summary: 283 layers, 7276605 parameters, 7276605 gradients, 17.1 GFLOPS

Downloading https://github.com/ultralytics/yolov5/releases/download/v6.0/yolov5s.pt to yolov5s.pt...
```

Reply:
v5.0 code are used, but v6.0 yolov5s.pt are downloaded.

Reply:
That's why v5.0 is always downloading the latest *pt https://github.com/ultralytics/yolov5/blob/f5b8f7d54c9fa69210da0177fec7ac2d9e4a627c/utils/google_utils.py#L25